### ISSUE TITLE
Error on Fivem Server 

### ISSUE THREAD

**Issue by SuryaRajgor** [ptitSeb/box64#324]
**Fivem Artifacts** : https://runtime.fivem.net/artifacts/fivem/build_proot_linux/master/5562-25984c7003de26d4a222e897a782bb1f22bebedd/fx.tar.xz

### **Console :**
Dynarec for ARM64, with extension: ASIMD AES CRC32 PMULL ATOMICS PageSize:4096
Box64 with Dynarec v0.1.9 6392550 built on Jun 11 2022 19:30:12
Using default BOX64_LD_LIBRARY_PATH: ./:lib/:lib64/:x86_64/:bin64/:libs64/
Using default BOX64_PATH: ./:bin/
Counted 25 Env var
Looking for /root/fivem/alpine/opt/cfx-server/ld-musl-x86_64.so.1
argv[1]="--library-path"
argv[2]="/root/fivem/alpine/usr/lib/v8/:/root/fivem/alpine/lib/:/root/fivem/alpine/usr/lib/"
argv[3]="--"
argv[4]="/root/fivem/alpine/opt/cfx-server/FXServer"
argv[5]="+set"
argv[6]="citizen_dir"
argv[7]="/root/fivem/alpine/opt/cfx-server/citizen/"
2157|SIGSEGV @0x6494cc10 (???(/usr/local/bin/box64+0x6494cc10)) (x64pc=0xffff99d61d8b//root/fivem/alpine/opt/cfx-server/ld-musl-x86_64.so.1:"???", rsp=0xffff99c87288, stack=0xffff99488000:0xffff99c88000 own=(nil) fp=(nil)), for accessing 0x13cc8 (code=1/prot=0), db=(nil)((nil):(nil)/(nil):(nil)/???:clean, hash:0/0) handler=(nil)
Segmentation fault (core dumped)

**Comment by MatStef132** [comment]
Same issues here, any updates on that?

**Comment by MatStef132** [comment]
[screen.log](https://github.com/ptitSeb/box64/files/9067826/screen.log)
My logs here

**Comment by ptitSeb** [comment]
Ok, so I have to check, but I see this
```
Rename process to "ld-musl-x86_64.so.1"
```
This will not work with box64. Box will not emulate `ld-musl` or `ld-linux`. I'll check to add a workaround to avoid trying to emulate this.

**Comment by SuryaRajgor** [comment]
https://pastebin.com/6za3VPKZ

**Comment by QuintenQVD0** [comment]
I have the same error with all builds:
https://pastebin.com/dViANHit
I used the debian repo to install box64

**Comment by ptitSeb** [comment]
I have added some `musl` support in box64, so the shell script can be used. Still musl support on box64 is *very* limited, so I don't expect things to run correctly for now.

**Comment by QuintenQVD0** [comment]
> I have added some `musl` support in box64, so the shell script can be used. Still musl support on box64 is _very_ limited, so I don't expect things to run correctly for now.

Atleast thx for the effort.  but It currently still does not start. So we will wait for some more updates!

**Comment by ptitSeb** [comment]
What is the expected output from that command?

**Comment by QuintenQVD0** [comment]
> What is the expected output from that server launch?

IT should start with:
![afbeelding](https://user-images.githubusercontent.com/67589015/180602122-b781d48e-4b38-4757-851a-c04c8b224cb2.png)

**Comment by MatStef132** [comment]
These are the logs for fivem (fxserver). Yeah, it still doesn't start
[screen.log](https://github.com/ptitSeb/box64/files/9275443/screen.log)

**Comment by QuintenQVD0** [comment]
@ptitSeb Do you still have any intrest of supporting / getting this game to run?

**Comment by ptitSeb** [comment]
I'm interested in running everything, but I don't have much time for now. And the musl use is a complicated issue.

**Comment by LV-Soft** [comment]
You can use **Box64** and **Wine** with _Windows_ artifacts,
I have not tested the performance, since a few months ago got banned from cfx.

![Screenshot_5](https://user-images.githubusercontent.com/12903132/183833132-45ba6ff8-4767-418d-a3f5-2b1abb8a6cd1.jpg)

**Comment by QuintenQVD0** [comment]
> You can use **Box64** and **Wine** with _Windows_ artifacts, I have not tested the performance, since a few months ago got banned from cfx.
> 
> ![Screenshot_5](https://user-images.githubusercontent.com/12903132/183833132-45ba6ff8-4767-418d-a3f5-2b1abb8a6cd1.jpg)

Can you please explain how you installed wine and what your startup cmd is? Can here or on discord `Quinten#6775`

**Comment by LV-Soft** [comment]
Yes, no problem... now I'm going to sleep a little. tomorrow I'll write you a text on discord detailing what I did, it's not complicated but I really messed up with this (not really with fxserver but with running a vrising server in arm64, but it worked for this).

**Comment by QuintenQVD0** [comment]
> Yes, no problem... now I'm going to sleep a little. tomorrow I'll write you a text on discord detailing what I did, it's not complicated but I really messed up with this (not really with fxserver but with running a vrising server in arm64, but it worked for this).

Thank you, have a good night.

**Comment by LukeShortCloud** [comment]
@SuryaRajgor @MatStef132 @LV-Soft @QuintenQVD0 Does the Linux version of the Fivem Server work with the latest version of Box64 now?

**Comment by QuintenQVD0** [comment]
It has been a while when I ran this but at that time no.
